tools/phenomenal/isatools/slicer/isa_get_study_factors.xml

```diff
@@ -1,28 +1,27 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <tool id="isa_get_study_factors" name="Get study factors" version="1.0">
     <description>
-        Get list of study factors for a study
+        Get list of study factors for a ISA-Tab study
     </description>
 
-    <requirements>
```

Comment:
There is no need to remove this requirements, in case you don't want to be maintaining two different versions of the wrapper.

Reply:
But up to you of course.